images/dockerregistry/Dockerfile

```diff
@@ -9,6 +9,9 @@ FROM openshift/origin-base
 RUN yum install -y tree findutils epel-release && \
     yum clean all
 
+# The registry doesn't require a privileged user.
+USER 1001
```

Comment:
how does this interact with uids for persistent storage (which registry is more likely to have than the deployer)